health/health.d/ml.conf

```diff
@@ -5,21 +5,19 @@
 
 # examples below are commented, you would need to uncomment and adjust as desired to enable them.
 
-# node level anomaly rate example
+# node level anomaly rate
 # https://learn.netdata.cloud/docs/agent/ml#node-anomaly-rate
-# if node level anomaly rate is between 1-5% then warning (pick your own threshold that works best via tial and error).
-# if node level anomaly rate is above 5% then critical (pick your own threshold that works best via tial and error).
-# template: ml_1min_node_ar
-#       on: anomaly_detection.anomaly_rate
-#       os: linux
-#    hosts: *
-#   lookup: average -1m foreach anomaly_rate
-#     calc: $this
-#    units: %
-#    every: 30s
-#     warn: $this > (($status >= $WARNING)  ? (1) : (5))
-#     crit: $this > (($status == $CRITICAL) ? (5) : (100))
-#     info: rolling 1min node level anomaly rate
+# if node level anomaly rate is above 1% then warning (pick your own threshold that works best via trial and error).
+template: ml_1min_node_ar
+      on: anomaly_detection.anomaly_rate
+      os: *
+   hosts: *
+  lookup: average -1m foreach anomaly_rate
```

Comment:
Do we need `foreach` here? This chart has only one dimension, right?

Reply:
Yeah only one dimension called `anomaly_rate`